app/components/transaction_table_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<table class="table table-bordered table-hover">
+  <thead>
+    <tr>
+      <th>Date</th>
+      <th>Project Number</th>
+      <th>Description</th>
+      <th>Amount</th>
+      <th>Installer</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% transactions.each do |transaction| %>
+      <tr>
+        <td><%= transaction.date.strftime("%m/%d/%y") %></td>
```

Comment:
`transaction.date.strftime("%m/%d/%y")` would be a good candidate for moving into the component class. Maybe even the link: `edit_transaction_installer_path(transaction, transaction.installer)`